documentation/policies.md

```diff
@@ -0,0 +1,15 @@
+## Policies
+
+* All major cloud providers (GCP, Azure, AWS) are treated with equal preference.  Some centralized services may be deployed on once cloud over another based on staff experise.
```

Comment:
> * All major cloud providers (GCP, Azure, AWS) are treated with equal preference.  Some centralized services may be deployed on ~once~ one cloud over another based on staff ~experise~ expertise.

Or something like:

* All major cloud providers (GCP, Azure, AWS) are treated with equal preference.  Some centralized services may be deployed on one cloud over another based on available resources such as staff expertise, cloud vendor support/implementation, or available credits.